### PR TITLE
Feat/wallet services and setupMFA

### PR DESCRIPTION
**Pull request by grvgoel81**
* Wallet services functionality
* SetUp MFA.
* Auth services v7 and v8 changes.

https://github.com/Web3Auth/web3auth-unity-sdk/assets/44719313/1bb182f3-ef66-497f-927d-be5af735aee3

https://github.com/Web3Auth/web3auth-unity-sdk/assets/44719313/ace12df2-9369-47d5-9c59-13f34fcf973e

### CODE REVIEW

File: Assets/Plugins/Web3AuthSDK/Web3Auth.cs [hunk 1]
```diff
@@ -310,13 +365,21 @@ public void setResultUrl(Uri uri)
         if (queryParameters.Keys.Contains("error"))
             throw new UnKnownException(queryParameters["error"]);
 
-        string sessionId = hash.Split('&')[0].Split('=')[1];
-
-        //save new sessionId
+        string b64Params = hash.Split('=')[1];
```
Comment (chaitanyapotti) [inline Assets/Plugins/Web3AuthSDK/Web3Auth.cs:368]: don't do this please

Comment (grvgoel81) [reply Assets/Plugins/Web3AuthSDK/Web3Auth.cs]: Updated

**Review by chaitanyapotti — APPROVED** [review]
lgtm

**Review by chaitanyapotti — APPROVED** [review]
lgtm

**Pull request merged**